west.yml

```diff
@@ -34,7 +34,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 236ec2d2553536835c76dff84adcb188309eaf90
+      revision: pull/98/head
```

Comment:
it seems to be pointing to a pull request not a sha
it does not seem correct...

Reply:
@pdunaj thanks for having a look. That should be correct: it will run CI against that PR. After that PR is in the revision will be updated to point to a SHA.

Reply:
This is correct.
As long as PR https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/98 is not merged, then revision `236ec2d2553536835c76dff84adcb188309eaf90` is not part of ref-space, and CI cannot run.

Thus the reason for `pull/98/head` to allow CI to fetch correct ref-space.